js/video/Video.stories.svelte

```diff
@@ -76,23 +76,7 @@
 	}}
 	play={async ({ canvasElement }) => {
 		const canvas = within(canvasElement);
-
 		const trimButton = canvas.getByLabelText("Trim video to selection");
-
 		userEvent.click(trimButton);
-
-		await new Promise((r) => setTimeout(r, 4000));
```

Comment:
This timeout was not sufficiently long enough for the element to appear, causing the storybook CI action to crash. @hannahblair I think we'll need to use a different approach.